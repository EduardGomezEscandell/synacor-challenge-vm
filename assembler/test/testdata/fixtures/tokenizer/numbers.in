0
16
0x15
0b011100
0x
0b
00
855
7855
0xbabe
0xgaga