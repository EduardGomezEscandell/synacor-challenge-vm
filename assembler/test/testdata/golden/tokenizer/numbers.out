<NUMBER 0> <EOL>
<NUMBER 16> <EOL>
<NUMBER 21> <EOL>
<NUMBER 28> <EOL>
<ERROR Unexpected end of digit in base-16 number literal>
<ERROR Unexpected end of digit in base-2 number literal>
<NUMBER 0> <EOL>
<NUMBER 855> <EOL>
<NUMBER 7855> <EOL>
<NUMBER 47806> <EOL>
<ERROR Unexpected digit in base-16 number literal: g>
<END>